PublicationsRouter/v2/sword-out/XWALK-DSpace.md

```diff
@@ -0,0 +1,115 @@
+# JPER Core Metadata to Dublin Core for DSPACE
+
+This document describes the crosswalk from the notification metadata fields received via the JPER API to the
+ XML format supplied to repositories via SWORDv2.
+ 
+For information about the schema see:
+
+* [Dubin Core](http://dublincore.org/documents/dcmi-terms/)
+
+The following table lists the notification fields in the JSON, then the DC terms fields it will be transformed to, and
+any notes regarding the transformation:
+
+| PubRouter Metadata | DC terms | Description |
+|:-----------------------------:|:-------------------------:|:------------------------------------------------------------:|
+| journal.title | bibliographicCitation |  |
+| journal.abbrevTitle | bibliographicCitation |  |
+| journal.volume | bibliographicCitation |  |
+| journal.issue | bibliographicCitation |  |
+| journal.publisher | publisher |  |
+| journal.identifier.issn | source  | (issn: value) |
+| journal.identifier.eissn | source | (eissn: value) |
+| journal.identifier.pissn | source  | (pissn: value) |
+| journal.identifier.doi | source | (doi: value) |
+| article.title | title | A name given to the resource. |
+| article.subtitle | title | A name given to the resource. |
+| article.type | type | The nature or genre of the resource. |
+| article.version | description |  |
+| article.start_page | bibliographicCitation |  |
+| article.end_page | bibliographicCitation |  |
+| article.page_range | bibliographicCitation |  |
+| article.num_pages | ---- |   |
+| article.language | language |  |
+| article.abstract | abstract |  |
+| article.identifier.type | identifier  | (type: id) |
+| article.identifier.id | identifier  | (type: id) |
+| article.subject | subject |  |
+| author.type | creator |  |
+| author.name | creator |  |
+| author.organisation_name | creator |  |
+| author.identifier.orcid | creator |  |
+| author.identifier.email | creator |  |
+| author.affiliation | ---- |   |
+| contributor.type | contributor |  |
+| contributor.name | contributor |  |
```

Comment:
In description give the name format e.g. Lastname, Forenames (if that is correct).